edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Hosting.cs

```diff
@@ -51,6 +51,8 @@ public class Hosting
                                         SslProtocols = sslProtocols
                                     });
                             });
+
+                        options.Listen(!Socket.OSSupportsIPv6 ? IPAddress.Any : IPAddress.IPv6Any, 9600);
```

Comment:
Is there ever a reason why we would want to change this port? I see that above, they are pulling a port from a configuration (with 443 as the default). Should you follow the same pattern?

Reply:
+1